.travis.yml

```diff
@@ -2,9 +2,9 @@
 
 language: rust
 rust:
-    - stable
```

Comment:
Why did you remove stable?

Reply:
We don't want to immediately update to stable when new versions are released because we're following [Firefox's policy](https://wiki.mozilla.org/Rust_Update_Policy_for_Firefox).